### DISCUSSION TITLE
how to solve validation on focus

### DISCUSSION THREAD

**Discussion by aditgege** [dm4t2/vue-currency-input#358] [Q&A]
Hi, I want to ask if it is possible to validate using a model value instead of a formatted value when not just focusing on the input element?

I have rules in component as follows
` :rules="[(val) => val > 0 || 'Must be greater than 0']"`

but when not focus on the element, it gets validated because the value passed to the rules is a formatted value
![image](https://user-images.githubusercontent.com/32119685/202610819-ba92d3b3-f222-41c3-9cd6-707665f9e912.png)

my current solution is make rules checking the value if contains a comma
`:rules="[(val) => val.includes(',') ? true : val > 0 || 'Must greater than 0' ]"`

is there a better solution, if the rules will be different later ? 

this is my repro
[codesandbox](https://codesandbox.io/s/vue-currency-input-integration-with-quasar-forked-o2gdqp?file=/src/QCurrencyInput.vue)

**Comment by dm4t2** [comment]
Hi, I would suggest this solution:
https://codesandbox.io/s/vue-currency-input-integration-with-quasar-gwn46?file=/src/QCurrencyInput.vue
